logger/modules/logger.py

```diff
@@ -0,0 +1,152 @@
+"""
+Logs debug messages.
+"""
+
+import datetime
+import inspect
+import logging
+import pathlib
+import os
+
+# Used in type annotation of logger parameters
+# pylint: disable-next=unused-import
+import types
+
+from read_yaml.modules import read_yaml
+
+
+CONFIG_FILE_PATH = pathlib.Path("config_logger.yaml")
```

Comment:
I have not checked, but I believe relative paths like this depend on where the command is executed. So similar to the relative imports issue, this would work if run from `py -m common...` (since you are in common folder) but will probably fail when running from `py -m modules.common...` (since you are in airside folder). Someone should test this, and I think using the `__file__` variable can help in this situation.